packages/storybook-html/src/components/card/card.docs.mdx

```diff
@@ -0,0 +1,116 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
+
+# Card
+
+<StatusBadge.Stable />
+
+<a href="https://github.com/uq-its-ss/design-system/issues" class="uq-button uq-button--text" target="_blank">Check for open issues</a>
+
+## Overview
+
+The card component is used to provide a brief summary of content or a task, often with a link to more detail. Cards are frequently displayed alongside other cards to group related content or tasks.
+
+## Default example
```

Comment:
I think your first example could be a group of three.. just to show the common usage.

Reply:
Done